src/main/java/com/violeth/blockjs/blockjs/CommandInterface.java

```diff
@@ -0,0 +1,36 @@
+package com.violeth.blockjs.blockjs;
+
+import com.violeth.blockjs.blockjs.jsinterface.JSRunner;
+import net.kyori.adventure.text.Component;
+import org.bukkit.command.CommandExecutor;
+import org.jetbrains.annotations.NotNull;
+
+public class CommandInterface implements CommandExecutor {
+    @Override
+    public boolean onCommand(
+        org.bukkit.command.@NotNull CommandSender sender,
+        org.bukkit.command.@NotNull Command command,
+        @NotNull String label, String[] args
+    ) {
+        if(args.length > 0) {
+            for(var i = 0; i < Commands.list.size(); i++) {
+                var commandInst = Commands.list.get(i);
+                if(args[0].equalsIgnoreCase(commandInst.key)) {
+                    commandInst.handle(sender, args);
+
+                    return true;
+                }
+            }
+
+            sender.sendMessage(BlockJS.getPluginMessagePrefix().append(
```

Comment:
I'm not entirely sure what the `BlockJS.getPluginMessagePrefix()` was meant to be, but no such method exists on the BlockJS class.